app/lib/definitions.ts

```diff
@@ -5,4 +5,14 @@ export type Screenings = {
   
 export type SpecificScreeningsResponse = {
     data: Screenings[];
+  };
+
+export type Settings = {
```

Comment:
I'm thinking you could change the name of this type to something like CarouselSettings because only "Settings" is very general and could also refer to other settings in the project

Reply:
it's fixed now, take a look and see that i've not missed anything